src/common/user_telemetry.rs

```diff
@@ -73,13 +76,19 @@ pub struct UserTelemetryConfigs {
     cluster_config: UserTelemetryClusterConfig,
 }
 
+#[derive(Serialize, Clone, Default)]
+pub struct UserTelemetryWebData {
+    responses: HashMap<usize, usize>,
```

Comment:
If the always key represents HTTP status code, maybe we could use a smaller type to represent it.
Maybe `u8` could be enough?